data/settings/src/main/java/com/google/jetpackcamera/settings/test/FakeJcaSettingsSerializer.kt

```diff
@@ -31,6 +36,14 @@ class FakeJcaSettingsSerializer(
     override val defaultValue: JcaSettings = JcaSettings.newBuilder()
         .setDarkModeStatus(DarkMode.DARK_MODE_SYSTEM)
         .setDefaultFrontCamera(false)
+        .setBackCameraAvailable(true)
+        .setFrontCameraAvailable(true)
+        .setFlashModeStatus(FlashMode.FLASH_MODE_OFF)
+        .setAspectRatioStatus(AspectRatio.ASPECT_RATIO_NINE_SIXTEEN)
+        .setCaptureModeStatus(CaptureMode.CAPTURE_MODE_MULTI_STREAM)
+        .setStabilizePreview(PreviewStabilization.PREVIEW_STABILIZATION_UNDEFINED)
+        .setStabilizeVideo(VideoStabilization.VIDEO_STABILIZATION_UNDEFINED)
+        .setStabilizationSupported(false)
```

Comment:
Instead of a `boolean`, can this be a list of the supported stabilization modes such as `ON` and `HIGH_QUALITY`?

Reply:
Ok- storing them as separate booleans is fine. I was just suggesting a list since it's easier to pass around as a single variable. I don't know if we need to convert them to a separate enum. In the settings composable can we just do something like:

```
Column(Modifier.selectableGroup()) {

  if (isPreviewStabilizationSupported) {
    // on selector
    SingleChoiceSelector(...)
  }

  if (isVideoStabilizationSupported) {
    // high quality selector
    SingleChoiceSelector(...)
  }

  // none selector
  SingleChoiceSelector(...)
}